app/Http/Controllers/Api/v1/Mensa/Controllers/SignupController.php

```diff
@@ -75,7 +77,12 @@ public function signup(Request $request, string $mensaId): ?JsonResponse
 
         $user = $this->currentUser();
         $isAdmin = $user != null && $user->mensa_admin;
-
+        
+        $log = new Log;
+        $log->category = "mensa";
```

Comment:
"mensa" doesn't look enforced, this seems very error prone. Maybe we could like move this to an enum or something?